style.css

```diff
@@ -1,9 +1,49 @@
 /* Write your CSS here! */
+/* body{
+  display: grid;
+}
+.sidebar{
+  grid-column: 1 / 2;
+  grid-row: 3 / 3;
+  align-content: left;
+} */
 
+.header{
+  grid-area: header;
+  
+}
+.menu{
+  padding-left: 350px;
+}
+img{
+  position: absolute;
+}
+.main {
+  grid-area: main;
+}
+.sidebar{
+  grid-area: sidebar;
+}
+.footer{
+  grid-area: footer;
+}
+
+
+body {
+  display: grid;
+  grid-template-columns: 100px 80px 1200px 100px;
```

Comment:
I think the proportions here on line 34 are a little off, but it still came out well. Nice job!